apps/sasquatch/src/androidTest/java/com/microsoft/azure/mobile/sasquatch/activities/SettingsActivityTest.java

```diff
@@ -0,0 +1,203 @@
+package com.microsoft.azure.mobile.sasquatch.activities;
```

Comment:
Also xamarin ui test is a separate app that provides all the label test screens to update.